cmd/recorder/main.go

```diff
@@ -42,10 +42,16 @@ func main() {
 	slog.Info("starting recording")
 
 	if err := recorder.Start(); err != nil {
+		slog.Error("failed to start recording", slog.String("err", err.Error()))
 		if err := recorder.ReportJobFailure(err.Error()); err != nil {
 			slog.Error("failed to report job failure", slog.String("err", err.Error()))
 		}
-		slog.Error("failed to start recording", slog.String("err", err.Error()))
+
+		// cleaning up
+		if err := recorder.Stop(); err != nil {
+			slog.Error("failed to stop recorder", slog.String("err", err.Error()))
+		}
```

Comment:
I wonder if we should surface these to the sysadmin somehow, or else they might get lost/ignored. I understand simply logging most of what's in the recorder (that's info you would go find when things go wrong), but this seems like something that might get missed.

Reply:
The job failed to start, that's a major problem. If stopping fails it's more of a minor issue at that point I am thinking.